book/src/themes.md

```diff
@@ -123,3 +123,24 @@ black = "#000000"
 
 Remember that the `[palette]` table includes all keys after its header,
 so you should define the palette after normal theme options.
+
+If there is no `[palette]` section, a default palette which uses the terminal's default 16 colors are used:
```

Comment:
Shouldn't it be merged? What if users want to use a palette as well as the default 16 colors?

Reply:
Ah you're right it's better to have merge the default one with the user palette (I can't remember the reason I decided against merging, but it seemed trivial anyway).